src/simulation/dynamics/FuelTank/fuelTank.cpp

```diff
@@ -132,6 +133,11 @@ void FuelTank::updateEffectorMassProps(double integTime) {
         (*fuelSloshInt)->fuelMassDot = (*fuelSloshInt)->massToTotalTankMassRatio * (-this->fuelConsumption);
     }
 
+    // - Set total fuel mass parameter for thruster dynamic effectors experiencing blow down effects
```

Comment:
No reason to have a hyphen `-` here in the comment.  Doxygen doesn't carry these comment lines into documentation, sadly.

Reply:
Agreed, removing all cases of unneeded hyphens.